src/seedu/addressbook/storage/Encryptor.java

```diff
@@ -0,0 +1,61 @@
+package seedu.addressbook.storage;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import javax.xml.bind.DatatypeConverter;
+
+/**
+ * Encrypts and decrypts a string using AES
+ */
+public class Encryptor {
+
+    private static String key = "0000000000000000"; // Placeholder key
```

Comment:
Placeholder key cannot be changed from outside the IDE/ on runtime. A future extension may be to allow each program to have their own secret key rather than all being the placeholder.